content/authors/Taves-mwangi/index.md

```diff
@@ -0,0 +1,6 @@
+---

```

Comment:
`content/authors/Taves-mwangi/index.md` -> `content/authors/taves-mwangi/index.md`

Change the article directory with all the letters in lowercase